bundles/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/ModbusGenericBindingProvider.java

```diff
@@ -152,14 +153,19 @@ State getItemState() {
             return item.getState();
         }
 
+        public void setItemState(State state) {
+            ((GenericItem) item).setState(state);
+            ;
```

Comment:
There is no problem holding onto a State object in your binding (in a special case like this where you can't be stateless).